basics/section_2.rb

```diff
@@ -0,0 +1,131 @@
+# 1 ~ 11
+# Section 1: Lets code the quiz!
+
+address = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+# var = address array is the rest
+
+p address
+# p = print
+
+p address[5]
+# print the number with index 5 
+
+new_address = address.reverse!
+# .reverse! is a method that comes with ruby
+
+p new_address
+# reversed list
```

Comment:
What is the value of `address` here?
Why is it so?

Reply:
[9,8, 7,6,5,4,3,2,1] because `reverse! ` also changes the original arrangement of the array.

Reply:
So, why do you need `new_address`?